packages/labs/gen-wrapper-react/test-output/package.json

```diff
@@ -71,7 +71,7 @@
         "build",
         "../../../tests:build"
       ],
-      "command": "node ../../../tests/run-web-tests.js \"tests/**/*_test.js\" --config ../../../tests/web-test-runner.config.js",
+      "command": "node ../../../tests/run-web-tests.js \"tests/**/(tests|*_test).js\" --config ../../../tests/web-test-runner.config.js",
```

Comment:
`tests.js` pulls in all the `_test` files, so doesn't this run them all twice? Why have the `tests.js` file?

Reply:
It's because we need an entrypoint for rollup? If so, then seems like that's the only thing that should be here?

Reply:
Fixed.